packages/node-cli/src/commands/CreateProjectCommandGenerator.js

```diff
@@ -402,35 +416,35 @@ module.exports = class CreateProjectCommandGenerator extends BaseCommandGenerato
 		}
 	}
 
-	_formatOutput(result) {
-		if (!result) {
+	_formatOutput(actionResult) {
+		if (!actionResult) {
 			return;
 		}
-		if (SDKOperationResultUtils.hasErrors(result.operationResult)) {
-			NodeUtils.println(
+		if (SDKActionResultUtils.hasErrors(actionResult)) {
+			logger.println(
```

Comment:
i think `logger` was one proposal but you forgot to change it back to `NodeUtils`

Reply:
yeah good catch!